app/views/shared/_ssn_field.html.erb

```diff
@@ -15,7 +15,7 @@ locals:
         required: true,
         pattern: '^\d{3}-?\d{2}-?\d{4}$',
         maxlength: 11,
-        input_html: { class: 'ssn-toggle usa-input', value: '' },
+        input_html: { class: 'ssn-toggle usa-input', value: f.object.ssn },
```

Comment:
Very happy to see use the form for this, since I think it means we can just let SimpleForm do its thing and not need to explicitly provide the value at all?

Also, fairly certain that `usa-input` should be assigned by default now.

```suggestion
        input_html: { class: 'ssn-toggle' },
```

Reply:
It looks like unfortunately I do have to specify the value because the underlying component does not appear to tie into simple form. It is used in a log of places so I am hesitant to change it here since that may get out of hand.

Reply:
Also, it looks like removing `usa-input` troubles the a11y specs (ref: https://gitlab.login.gov/lg/identity-idp/-/jobs/413342)

Reply:
Your test failure might have been a flake, because when I check out that commit and run in locally it passes. I can also confirm upon inspecting the page that the `usa-input` class is in there already by default

Reply:
Just as a data point, I reverted 2b26e7f18e1db19dcbd69f86856f220f65656961 and ran `spec/features/accessibility` locally and it passed a-ok

Reply:
Hmm, lemme try again

Reply:
> It looks like unfortunately I do have to specify the value because the underlying component does not appear to tie into simple form.

Not sure if it's what you meant by underlying component, but I spent some time digging into this since I thought it should "just work". Turns out that SimpleForm eventually delegates to Rails' built-in `password_field` tag builder, which sets the value to `nil` by default.

https://github.com/rails/rails/blob/v7.0.4.3/actionview/lib/action_view/helpers/tags/password_field.rb#L8

For non-password fields, it probably would work fine without the explicit `value` to use the value from the SimpleForm object.